server/auth.ts

```
@@ -2,63 +1,63 @@
-      return
-    }
-    assertIsDefined(decripted)
     if (
       shallowEqualScalar(
         author,
         deleteJWTattribute(decripted) as IndexSignature<JWTpayload>,
       )
     ) {
       // Verified
-      return true
+      return next()
```

Biome (1.9.4)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)